tests/test_tsp.py

```diff
@@ -97,6 +97,29 @@ def test_start(self):
 
         self.assertEqual(route[0], 1)
 
+    def test_weighted_complete_digraph(self):
+        G = nx.DiGraph()
+        G.add_weighted_edges_from([
+            (0, 1, 2),
+            (1, 0, 1),
+            (0, 2, 2),
+            (2, 0, 2),
+            (0, 3, 1),
+            (3, 0, 2),
+            (1, 2, 2),
+            (2, 1, 1),
+            (1, 3, 2),
+            (3, 1, 2),
+            (2, 3, 2),
+            (3, 2, 1),
+        ])
+
+        route = dnx.traveling_salesperson(G, dimod.ExactSolver(), start=1)
+
+        self.assertEqual(len(route), len(G))
+        self.assertEqual(nx.path_weight(G, route, 'weight'), 4)
```

Comment:
Looks like this function requires a newer networkx than the minimum we support. I would be OK with bumping up the min networkx version in `setup.py`, but given that it's just for a test it might make sense to change the test a bit.